Assets/Engine/Inventory/AttachedContainer.cs

```diff
@@ -4,6 +4,9 @@
 using Mirror;
 using SS3D.Content;
 using UnityEngine;
+using SS3D.Content.Systems.Examine.Examinables;
```

Comment:
Please remove this line. Referencing Content is no longer necessary - everything you have done is with the Examine system. That's exactly where we want to be - we can change the Content as much as we like, and your code will still work as long as the System hasn't been altered.

Reply:
Done